frontends/web/src/routes/receive/receive.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { IAccount } from '../../api/account';
+import { AccountSelector } from '../../components/accountSelector/accountselector';
+import { Guide } from '../../components/guide/guide';
+import { Entry } from '../../components/guide/entry';
+import { Header } from '../../components/layout';
+import { route } from '../../utils/route';
+import { isBitcoinOnly } from '../account/utils';
+
+type TReceiveAccountsSelector = {
+    activeAccounts: IAccount[]
+}
+export const ReceiveAccountsSelector = ({ activeAccounts }: TReceiveAccountsSelector) => {
+  const [code, setCode] = useState('');
+
+  const { t } = useTranslation();
+
+  const handleProceed = () => {
+    route(`/account/${code}/receive`);
+  };
+
+  const options = activeAccounts.map(account => ({ text: account.name, value: account.code }));
+
+  const hasOnlyBTCAccounts = activeAccounts.every(({ coinCode }) => isBitcoinOnly(coinCode));
+
+  const title = t('receive.title', { accountName: hasOnlyBTCAccounts ? 'Bitcoin' : 'crypto' });
+
+  return (
+    <div className="contentWithGuide">
+      <div className="container">
+        <Header title={<h2>{title}</h2>} />
+        <div className="innerContainer scrollableContainer">
+          <AccountSelector onChange={setCode} onProceed={handleProceed} options={options} title={title} selected={code} />
+        </div>
+      </div>
+      <Guide>
+        <Entry key="guide.receive.plugout" entry={t('guide.receive.plugout')} />
+      </Guide>
```

Comment:
One thing, I included a Guide here. I took it from the individual receive page (`accounts/${code}/receive`). 

It can (probably should) be changed later on, just to make sure we have guide included here.

Feel free to lmk if we have Guide content ready for this new receive page, and I can put them here. :)

Reply:
No guide needed (for now=.